internal/controller/instaslice_controller.go

```diff
@@ -71,11 +71,26 @@ func (r *InstasliceReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	policy := &FirstFitPolicy{}
 	pod := &v1.Pod{}
+	var instasliceList inferencev1alpha1.InstasliceList
+	if err := r.List(ctx, &instasliceList, &client.ListOptions{}); err != nil {
+		log.FromContext(ctx).Error(err, "Error listing Instaslice")
+	}
 	err := r.Get(ctx, req.NamespacedName, pod)
 	if err != nil {
 		// Error fetching the Pod
 		if errors.IsNotFound(err) {
-			log.FromContext(ctx).Error(err, "unable to fetch pod might be deleted")
+			log.FromContext(ctx).Info("unable to fetch pod might be deleted")
+			// TODO figure out why are allocations present post pod deletes?
```

Comment:
Do we have a reproducer of this issue? Maybe we can document it in detail in a github issue and refer it here in the TODO.

Reply:
Yes thanks, https://github.com/openshift/instaslice-operator/issues/150

Reply:
Thanks, can you add a comment with the issue link? e.g. https://github.com/kubernetes/kubernetes/blob/aa09157014750871407aca4217f4ecf07bc64b7c/pkg/kubelet/cadvisor/util.go#L74